prepare-image.sh

```diff
@@ -46,8 +47,16 @@ if [[ ! -z ${EXTRA_PKGS_LIST:-} ]]; then
     fi
 fi
 
+dnf install -y \
+    https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm \
+    https://dl.fedoraproject.org/pub/epel/epel-next-release-latest-9.noarch.rpm
+dnf config-manager --set-disabled epel epel-next
 dnf install -y --enablerepo=epel inotify-tools
```

Comment:
I wonder if we need to enable epel-next here. Otherwise why do we install it at all? This is the only place where EPEL is used.

Reply:
I Ieft this here because at some point there may be other libraries coming from epel-next, but we don't need this now, I will just remove it for the time being